conf/test.config

```diff
@@ -22,15 +22,16 @@ params {
     // Input data
     // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
     // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input                         = 'https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/samplesheet.csv'
-    databases                     = 'https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/database.csv'
-    run_kraken2                   = true
-    run_malt                      = true
-    run_metaphlan3                = true
-    run_centrifuge                = true
+    input                                 = 'https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/samplesheet.csv'
+    databases                             = 'https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/database.csv'
+    run_kraken2                           = true
+    run_malt                              = true
+    run_metaphlan3                        = true
+    run_centrifuge                        = true
     perform_shortread_clipmerge           = true
     perform_longread_clip                 = false
     perform_shortread_complexityfilter    = true
     perform_shortread_hostremoval         = true
     shortread_hostremoval_reference       = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/genome/genome.fasta'
+    run_kaiju                             = true
```

Comment:
Could you move the other `run_` to the bototm as well? That would make more sense

Reply:
I resolve this as well since I moved the other `run_ `to the bottom